airgun/entities/contenthost.py

```diff
@@ -55,12 +59,44 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version):
+        """Execute remote module_stream action on a content
+
+        :param entity_name: content host name to remotely execute package
+            action on
+        :param action_type: remote action to execute. This is dict with containing 'Action'
+            and 'is_Customize' keys. Action key value can be one of
+            5: 'Enable', 'Disable', 'Install', 'Update', 'Remove', 'Reset'
+        :param module_name: Module Stream name to remotely
+            install/upgrade/remove (depending on `action_type`)
+        :param stream_version: this uniquely identifies the module with Stream Version
+
+        :return: Returns a dict containing job status details
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.module_streams.search('name = {} and stream = {}'.format(module_name, stream_version))
+        view.module_streams.table.row(
+            name=module_name, stream=stream_version)['Actions'].fill(action_type)
+        if isinstance(action_type, dict):
+            if 'is_customize' in action_type and action_type['is_customize']:
+                view = JobInvocationCreateView(view.browser)
+                view.submit.click()
+        view = JobInvocationStatusView(view.browser)
+        view.wait_for_result()
```

Comment:
Here you wait for up to 600 second, but in `entities/hostcollection.py` you wait for up to 60 seconds. Why is there a difference? Can you make them consistent?

Reply:
Yes, just missed that. I will update in both location,